src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py

```diff
@@ -2862,7 +2862,6 @@ def forward(self, *args, **kwargs):
         return self.decoder(*args, **kwargs)
 
 
-# Copied from transformers.models.bart.modeling_bart.BartForCausalLM with Bart->BigBirdPegasus, 'facebook/bart-large'->"google/bigbird-pegasus-large-arxiv"
```

Comment:
can't keep copy here, because there is no `BigBirdPegasusTokenizer`

Reply:
could we try to keep the copy and change `Bart->BigBirdPegasus` to `BartDecoderWrapper->BigBirdPegasusDecoderWrapper` and `BartForCausalLM->BigBirdPegasusForCausalLM`? This way the tokenizer should stay the same :-)

Reply:
Let me try it.

Reply:
The following does the trick:

- BartDecoderWrapper->BigBirdPegasusDecoderWrapper
- BartForCausalLM->BigBirdPegasusForCausalLM
- BartPreTrainedModel->BigBirdPegasusPreTrainedModel
- BartTokenizer->PegasusTokenizer
- 'facebook/bart-large'->"google/bigbird-pegasus-large-arxiv"

Reply:
perfect!